src/main/java/at/hannibal2/skyhanni/config/core/config/CustomColor.kt

```diff
@@ -0,0 +1,95 @@
+package at.hannibal2.skyhanni.config.core.config
+
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import java.awt.Color
+
+class CustomColor {
```

Comment:
Is this class not largely equivalent to https://github.com/NotEnoughUpdates/MoulConfig/blob/e594eecea139c7ad4e50f95707b6535b2ba535b1/common/src/main/java/io/github/notenoughupdates/moulconfig/ChromaColour.java which already should work with moulconfig out of the box